src/elements/teaser-product/teaser-product-static/readme.md

```diff
@@ -15,15 +17,21 @@ otherwise, see [sbb-teaser-product](/docs/elements-sbb-teaser-sbb-teaser-product
 
 ## Slots
 
-Use the `image` slot to pass a `sbb-image` or an `img` that will be used as a background,
-and use the optional `footnote` slot to add a text anchored to the bottom-end of the component.
+Use the `image` slot to pass a `figure` containing an `sbb-image` or an `img` that will be used as background.
```

Comment:
Please describe when a figure is needed (only if figcaption or chip-label is present)